### PR TITLE
Records an error if the excel file it is searching for doesn't exist

### LINKED ISSUE #196: Pull Issues with Excel
image

If the user hasn't exported the excel file, Excel will open anyway, but fail to close.
Excel is being opened as a background process when the excel file doesn't exist.

You need to re-add a way to check that a valid excel file type is being used to open excel as a background process.

checkFileExists = System.IO.File.exist(FilePath.xls)

if (checkFileExists){
        ##DO EXCEL STUFF
}
else {
        ##Throw custom exception "You need to export results from Midas to Excel"
}

### PR DESCRIPTION
**Pull request by calumlockhart-bh**
This will fix the problem for now, recommend that a more effective solution would be to also filter by file type to make sure that it is an excel file type being inserted into the function.

  
### Issues addressed by this PR

Closes #196 

### Test files

https://burohappold.sharepoint.com/:f:/s/BHoM/ElUYnAEfM75CmIYz7SvR_nIBQVzFj7681FMOmP2XOYlmHQ?e=acWWof

### CODE REVIEW

File: MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs [hunk 1]
```diff
@@ -196,28 +196,35 @@ private List<string> GetLoadcaseIDs(IResultRequest request)
...
-                    Type.Missing,
-                    Microsoft.Office.Interop.Excel.XlSaveAsAccessMode.xlShared,
-                    Type.Missing,
-                    Type.Missing,
-                    Type.Missing
-                );
-
-                xlsFile.Close();
+                Engine.Reflection.Compute.RecordError("MidasCivil_Toolkit detects no Excel file, make sure you have exported the results from Midas.");
+            }
```
Comment (peterjamesnugent) [inline MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs]: Can you 

`return null;`

So a value is returned for both instances of the `if` statement.

File: MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs [hunk 2]
```diff
@@ -196,28 +196,35 @@ private List<string> GetLoadcaseIDs(IResultRequest request)
...
-                    Type.Missing,
-                    Type.Missing,
-                    Microsoft.Office.Interop.Excel.XlSaveAsAccessMode.xlShared,
-                    Type.Missing,
-                    Type.Missing,
-                    Type.Missing
-                );
-
-                xlsFile.Close();
+                Engine.Reflection.Compute.RecordError("MidasCivil_Toolkit detects no Excel file, make sure you have exported the results from Midas.");
```
[the remark this answers is not in the document]
Comment (calumlockhart-bh) [reply MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs]: done

File: MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs (hunk 1 above)
Comment (calumlockhart-bh) [reply MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs]: done

**Pull request merged**